### DISCUSSION TITLE
Adapter Launch file questions - units etc.

### DISCUSSION THREAD

**Discussion by albertleng** [osrf/rmf_demos#215] [Q&A] [answered]
I am referring to launch files in [adapters](https://github.com/osrf/rmf_demos/tree/master/demos/launch/include/adapters).  

My team  is trying to find the definitions of each of these values, but I was unable to find good documentation describing what each of these field mean. Do you have documentation describing these fields? units?

The first section discussing linear and angular velocity and accelerations, I am assuming they want the gains and not the maximum values. Please correct me if I am wrong.

Thanks.

**Comment by Yadunund** [comment, marked as answer]
Hi @albertleng,

The description for various parameters in those launch files can be found [here](https://github.com/osrf/rmf_core/blob/6968346f40421d93039d898acdcbc3ca21f9ceeb/rmf_fleet_adapter/launch/fleet_adapter.launch.xml#L7-L39)

Hoping the answer to your second question becomes clear from reading the above. Let us know otherwise.
